scripts/author.js

```diff
@@ -20,34 +20,27 @@ if (!process.env.SEED) {
 }
 
 async function run() {
-  const wallet = await await Wallet.create({
-    ceramic,
-    seed: fromString(process.env.SEED),
-    getPermission() {
-      return Promise.resolve([])
-    },
-    disableIDX: true,
-  })
-  const did = new DID({ provider: wallet.getDidProvider() })
+  const seed = fromString(process.env.SEED, 'base16')
+  const did = new DID({ provider: new Ed25519Provider(seed), resolver: KeyResolver.getResolver() })
```

Comment:
Nice 👍

Reply:
Yeah it's great to be able to do without IDW in this setup now!